timeseries/src/autogluon/timeseries/models/chronos/model.py

```diff
@@ -265,6 +265,15 @@ def _is_gpu_available(self) -> bool:
 
         return torch.cuda.is_available()
 
+    @property
+    def model_pipeline(self) -> Any:  # of type BaseChronosPipeline
+        """The model pipeline used for inference. If the model is not loaded, this will be None."""
+        if self._model_pipeline is None:
+            # FIXME: optimization_strategy is ignored when model is fine-tuned
```

Comment:
Should we remove this FIXME since we're deprecating `optimization_strategy` anyway?